### PR TITLE
fix: similarity view to use feature tag

### PR DESCRIPTION
**Pull request by billyvg**

### CODE REVIEW

**Review by MaxBittker — APPROVED** [review]
this looks good to me, but the spread + ternary is bordering on overly clever :P

**Comment by MaxBittker** [comment]
would be nice if spreading a falsey value worked like spreading []

File: src/sentry/static/sentry/app/views/groupGrouping/groupGroupingView.jsx [hunk 1]
```diff
@@ -119,8 +125,10 @@ const GroupGroupingView = React.createClass({
     let {orgId, projectId, groupId} = this.props.params;
     let isLoading = this.state.loading;
     let isError = this.state.error && !isLoading;
+    let projectFeatures = this.getProjectFeatures();
     let hasMergedItems = this.state.mergedItems.length >= 0 && !isError && !isLoading;
     let hasSimilarItems =
+      projectFeatures.has('similarity-view') &&
       (this.state.similarItems.length >= 0 ||
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/groupGrouping/groupGroupingView.jsx]: that's a lot of conditions

**Comment by billyvg** [comment]
You're right, the spread was being too lazy and trying to do it inline, updated.

**Pull request merged**